src/views/Pools/components/LockedPool/Common/BalanceField.tsx

```diff
@@ -33,6 +34,13 @@ const BalanceField: React.FC<PropsType> = ({
 }) => {
   const { t } = useTranslation()
   const [percent, setPercent] = useState(0)
+  const { balance: userCake, fetchStatus } = useTokenBalance(tokens.cake.address)
```

Comment:
Hi @Chef-Yogi  !

Could you wrap this new logic into a new hook like 
```
const { userNotEnoughCake, notEnoughErrorMessage } = useUserEnoughValidator(...)
```

It would make the code more easier to maintain.

Reply:
sure